enshrine-esdt-safe/src/to_sovereign/create_tx.rs

```diff
@@ -111,8 +111,8 @@ pub trait CreateTxModule:
             return;
         }
 
-        let all_names = self.verify_items_signature(opt_signature, names);
-        self.remove_items(&mut self.banned_endpoint_names(), &all_names);
+        // let all_names = self.verify_bls_signature(opt_signature, names);
+        // self.remove_items(&mut self.banned_endpoint_names(), &all_names);
```

Comment:
commented code

Reply:
this code will be changed once the the `feat/enshrine` branch will be merged into this one, the commented code is a really old version that will be replaced